config/permissions.yml

```diff
@@ -1340,6 +1340,11 @@ feature_organisation_set_showlist:
     label: 'Anzeigen der Organisation in der Institutionen-Liste'
     loginRequired: true
     parent: [area_manage_orgas, area_manage_orgas_all]
+feature_organisation_user_list:
+    description: 'Show Organisations fields in use list'
```

Comment:
may be we have to explain why we need this new permission, something like 'unlike the 'area_organisations' permission, the 'feature_organisation_user_list' permission allows to show Organisations list but does not allow access to the 'Organisation' item in the menu.

Reply:
done in [175abdd](https://github.com/demos-europe/demosplan-core/pull/3475/commits/175abddb14582d545653d01d87448040e005a1f8)